library/lib/application.cpp

```diff
@@ -353,8 +427,64 @@ bool Application::mainLoop()
         Application::exit();
         return false;
     }
+    // Poll Touches from the switch's touch screen.
```

Comment:
I think glfw switch already binds touch to glfw mouse :eyes: Can you update your switch-glfw package and make sure it's the case?

We'll get rid of glfw for switch eventually so it's actually good that you made that (cc @WerWolv )

Reply:
Yea I was initially going down that route and using mouse implementation in glfw to make the touch-screen work, but there were some issues I came across that made me just go ahead and use the hid touch screen functions provided by libnx.

But what's the reasoning behind getting rid of glfw? Was it meant to be temporary?

Reply:
The issue with glfw on the switch is that it's massive. Switching to deko means cutting down the executable size from like 7MB to about 500kB. This is great for homebrew but even better for when people eventually try to reimplement for example applets. I did some tests with borealis replacing qlaunch and it wouldn't even load the example without at least 150MB of heap allocated. This will also be resolved with deko.